src/System.Private.CoreLib/shared/System/Double.cs

```diff
@@ -319,7 +358,7 @@ bool IConvertible.ToBoolean(IFormatProvider provider)
 
         char IConvertible.ToChar(IFormatProvider provider)
         {
-            throw new InvalidCastException(String.Format(SR.InvalidCast_FromTo, "Double", "Char"));
+            throw new InvalidCastException(SR.Format(SR.InvalidCast_FromTo, "Double", "Char"));
```

Comment:
`SR.Format` is the right way to format exception messages. (It does the right thing when the resource strings are stripped - see https://github.com/dotnet/corefx/pull/3078 .)